extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -136,32 +119,32 @@ describes.realWin('Bind', {
   it('should throw error if experiment is not enabled', () => {
     toggleExperiment(env.win, 'amp-bind', false);
     // Experiment check is bypassed on test mode -- make sure it isn't.
-    window.AMP_MODE = {test: false};
+    env.sandbox.stub(getMode(), 'test', false);
```

Comment:
Setting `window.AMP_MODE` is more common but stubbing avoids overwriting other mode properties (e.g. line 180). 

IMO stubbing is generally better practice than setting global vars, but maybe there's something I'm missing. /cc @jridgewell